.github/workflows/test-deployment.yml

```diff
@@ -0,0 +1,129 @@
+name: Test deployment 
```

Comment:
What do you think about renaming this to `test-release`?

No deployment is actually simulated, only versioning and releasing (ie: no assets are uploaded, etc.)